### ISSUE TITLE
Terminal window resize may crash the program

### ISSUE THREAD

**Issue by MichalLebeda** [gin66/tui-logger#60]
Quickly resizing from big window to small (~25 chars width) crashes the program. Tested using Kitty terminal.
```
thread 'main' panicked at /home/.../tui-logger-0.11.0/src/lib.rs:1203:33:
attempt to subtract with overflow
```

**Comment by gin66** [comment]
Thanks. Fix applied in 0.11.1

**Comment by MichalLebeda** [comment]
Thank you so much!
